docs/self-managed/setup/deploy/amazon/amazon-eks/terraform-setup.md

```diff
@@ -33,203 +41,171 @@ To try out Camunda 8 or develop against it, consider signing up for our [SaaS of
 
-### Cluster module
+2. **Run the following command** to create the S3 bucket, using the previously set `AWS_REGION` environment variable:
 
-This module creates the basic layout that configures AWS access and Terraform.
+   ```bash
+   aws s3api create-bucket --bucket my-eks-tf-state --region $AWS_REGION \
+     --create-bucket-configuration LocationConstraint=$AWS_REGION
+   ```
 
-The following will use [Terraform modules](https://developer.hashicorp.com/terraform/language/modules), which allows abstracting resources into reusable components.
+3. **Enable versioning** on the S3 bucket to track changes and protect the state file from accidental deletions or overwrites:
 
-The [Camunda provided module](https://github.com/camunda/camunda-tf-eks-module/tree/2.5.0/modules/eks-cluster) is publicly available. It's advisable to review this module before usage.
+   ```bash
+   aws s3api put-bucket-versioning --bucket my-eks-tf-state --versioning-configuration Status=Enabled --region $AWS_REGION
+   ```
 
-1. In the folder where your `config.tf` resides, create an additional `cluster.tf`.
-2. Paste the following content into the newly created `cluster.tf` file to make use of the provided module:
+4. **Secure the bucket** by blocking public access:
 
-```hcl
-module "eks_cluster" {
-  source = "git::https://github.com/camunda/camunda-tf-eks-module//modules/eks-cluster?ref=2.5.0"
+   ```bash
+   aws s3api put-public-access-block --bucket my-eks-tf-state --public-access-block-configuration \
+     "BlockPublicAcls=true,IgnorePublicAcls=true,BlockPublicPolicy=true,RestrictPublicBuckets=true" --region $AWS_REGION
+   ```
 
-  region  = "eu-central-1" # change to your AWS region
-  name    = "cluster-name" # change to name of your choosing
+5. **Verify versioning** is enabled on the bucket:
 
-  # Set CIDR ranges or use the defaults
-  cluster_service_ipv4_cidr = "10.190.0.0/16"
-  cluster_node_ipv4_cidr    = "10.192.0.0/16"
-}
-```
+   ```bash
+   aws s3api get-bucket-versioning --bucket my-eks-tf-state --region $AWS_REGION
+   ```
 
-There are various other input options to customize the cluster setup further; see the [module documentation](https://github.com/camunda/camunda-tf-eks-module/tree/2.5.0/modules/eks-cluster).
+This S3 bucket will now securely store your Terraform state files with versioning enabled.
 
-### PostgreSQL module
+#### 2. Create a `config.tf` with the Following Setup
 
-The resulting PostgreSQL instance and default database `camunda` is intended to be used with Keycloak. You may manually add extra databases after creation for Identity with multi-tenancy.
-This will not be covered in this guide as the Identity default for multi-tenancy is to be disabled.
+Once the S3 bucket is created, configure your `config.tf` file to use the S3 backend for managing the Terraform state:
 
-We separated the cluster and PostgreSQL modules from each other to allow more customization options to the user.
+<Tabs>
```

Comment:
You maybe want to mention somewhere in the doc, probably considerations or so, what the difference is between the two options.